index.html

```diff
@@ -42,5 +42,175 @@ <h1 class="name"> Hey there <br>
           <img src="images/Disabled.svg" alt="drop-down">
         </div>
     </section>
+    <div class="line-break">
+        <h2 class="works">My Recent <br> Works</h2>
+        </div>
+      <section class="section-placeholder">
+        <div  class="pictureholder-container">
+            <img src="Images 2/placeholder.svg" alt="placeholder" class="placeholder">
+            <div class="text-container">
+              <h5  class="header2">Multi-Post Stories <br> Gain+Glory</h5>
+              <ul class="text">
+                <li class="text-ruby">Ruby on Rails</li>
+                <li class="text-css">CSS</li>
+                <li class="text-java">JavaScript</li>
+                <li class="text-html">HTML</li>
+              </ul>
+              <div class="button1"><button class="button-text" type="button">See project </button></div>
```

Comment:
your cards looks good I have only one small thing to mention is the margin between list items of technologies and  also the padding if you increase the padding you will make it look exactly like the design 

![image (1)](https://github.com/Wils-FOURR-speed/Portfolio-setup-and-mobile-first-project/assets/59707859/a3089b75-3600-4d25-b8e2-d03718e97b68)
![image (2)](https://github.com/Wils-FOURR-speed/Portfolio-setup-and-mobile-first-project/assets/59707859/8cec6c4f-e137-41da-a25a-2a8dfa1c2e5d)